<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_History</class>
 <widget class="QDialog" name="Dialog_History">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>173</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cleaning History</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rrcc.qrc">
    <normaloff>:/png/png/history.png</normaloff>:/png/png/history.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Details</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_state">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../rrcc.qrc">:/png/png/canceled.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../rrcc.qrc">:/png/png/start.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLCDNumber" name="lcdNumber_start">
          <property name="digitCount">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../rrcc.qrc">:/png/png/stop.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLCDNumber" name="lcdNumber_stop">
          <property name="digitCount">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../rrcc.qrc">:/png/png/clock.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLCDNumber" name="lcdNumber_duration">
          <property name="digitCount">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../rrcc.qrc">:/png/png/compass.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLCDNumber" name="lcdNumber_area">
          <property name="digitCount">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_error_icon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../rrcc.qrc">:/png/png/warning.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_error_text">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="text">
           <string notr="true">?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rrcc.qrc"/>
 </resources>
 <connections/>
</ui>
